README.md

```diff
@@ -1,7 +1,64 @@
 # API Compare Tool
 
-A tool that can be used to compare two versions of a class API.
+A tool that can be used to compare two versions of a class API in PHP
+code.
 
-## Installation & Usage 
+## Pre-requisites/assumptions
 
-TBD - still in development
+ * Your project uses `git`
+ * You use semver formatted `git` tags like `1.2.3` to mark releases
```

Comment:
Do we support parsing things like `release-1.2.3` or `v1.2.3`?

Reply:
For now at least, no. Issue maybe?